CHANGES.rst

```diff
@@ -1,6 +1,12 @@
 Changes
 =======
 
+Version 0.3.0 (2018-08-10)
+--------------------------
+
+- Introduces the ``reana`` command line tool for developers.
```

Comment:
Here we should display a short user-oriented overview of new important features since v0.2.0.  WIP:

- Introduces new Serial workflow engine for simple sequential parametrised workflow needs.
- Introduces multi-user capabilities with mandatory access tokens. 
- Enhances workflow progress reporting for CWL, Serial and Yadage workflow engines.
- Simplifies ``reana-client`` command set structure and removes distinction between inputs code and data.
- Fixes several bugs such as binary file download with Python 3.
- Adds support for Kubernetes 1.11, Minikube 0.28.2.